Makefile

```diff
@@ -0,0 +1,40 @@
+HOST = localhost:4000
+NPM_BIN = ./node_modules/.bin
+export PATH := $(NPM_BIN):$(PATH)
+
+# Install dependencies
+install:
+	@echo "Installing Pa11y..."
+	@npm install pa11y@^3
+	@echo "Installing GitHub Pages gem..."
+	@gem install github-pages
+
+# Build the site
+build:
```

Comment:
Ok cool, I've not used `make` before so was mainly just checking whether these would be useful outside of Circle. In which case, nice addition 👍